BUILD.md

```diff
@@ -46,48 +45,48 @@
     * `make install -j4`
 
 14. Go to **libusbmuxd** folder
-    * `cd /home/libusbmuxd-9db5747cd823b1f59794f81560a4af22a031f5c9/`
+    * `cd /home/libusbmuxd-b097ea39f391f5c2c83d8f4687843a3634f7cd54/`
 
 15. Build library
     * `./autogen.sh`
     * `make install -j4`
 
 16. Go to **openssl** folder
-    * `cd /home/openssl-OpenSSL_1_1_0`
+    * `cd /home/openssl-OpenSSL_1_1_1`
 
 17. Build library
     * `./Configure --prefix=/mingw32 no-idea no-mdc2 no-rc5 shared mingw`
     * `make -j4`
     * `make install_sw -j4`
 
 18. Go to **libimobiledevice** folder
-    * `cd /home/libimobiledevice-92c5462adef87b1e577b8557b6b9c64d5a089544/`
+    * `cd /home/libimobiledevice-a9e69b1252e5918b6d8ada1209ccefde301cfa26/`
 
 19. Build library
     * `./autogen.sh --without-cython`
     * `make install -j4`
 
 20. Go to **pcre** folder
-    * `cd /home/pcre-8.41/`
+    * `cd /home/pcre-8.43/`
 
 21. Build library
     * `./configure`
     * `make install -j4`
 
 22. Go to **ios-webkit-debug-proxy** folder
-    * `cd /home/ios-webkit-debug-proxy-1.8/`
+    * `cd /home/ios-webkit-debug-proxy-outsystems/`
 
 _**Warning:** If you are using OutSystems Github fork you don’t need to apply these changes and you can skip to Build library step._
 
 23. Edit **socket_manager.c**
-    * Open `C:\msys32\home\ios-webkit-debug-proxy-1.8\src\socket_manager.c`
-    * Edit line 90 from:
+    * Open `C:\msys32\home\ios-webkit-debug-proxy-outsystems\src\socket_manager.c`
+    * Edit line 120 from:
```

Comment:
Commit this on the branch and remove it from the instructions

Reply:
Done :) Thanks

Reply:
btw, good catch 💪